src/masonite/exceptions/ExceptionHandler.py

```diff
@@ -29,6 +29,9 @@ def get_config_options(self, driver=None):
     def handle(self, exception):
         response = self.application.make("response")
         request = self.application.make("request")
+        self.application.make("event").fire(
```

Comment:
@girardinsamuel 
So here it means that every exception caught by the handler is fired as an event with the name of the exception ? nice 💪